packages/firestore/src/remote/serializer.ts

```diff
@@ -644,6 +635,16 @@ export function toMutation(
     return fail('Unknown mutation type ' + mutation.type);
   }
 
+  if (
+    mutation instanceof MutationWithTransforms &&
+    mutation.fieldTransforms &&
+    mutation.fieldTransforms.length > 0
```

Comment:
Again, this could be simplified a bit. We should make sure though to not send an empty array if we no transforms exist (instead, we should just not specify the field to save on bandwidth).